### PR TITLE
Fix broken link to Godot Recipes text tutorial

### PR DESCRIPTION
**Pull request by cvlnomen**

### CODE REVIEW

**Comment by mhilbrunner** [comment]
That URL seems to work for me.

**Comment by cvlnomen** [comment]
Ah, I understand what is going wrong: the website does not support https, and my browser blocks http pages.
When the godotrecipes.com URL can be loaded successfully, the website will redirect the browser to kidscancode.org/godot_recipes. I think that's why it's still a good idea to change the url.

File: community/tutorials.rst [hunk 1]
```diff
@@ -50,7 +50,7 @@ Text tutorials
 
 - `FinepointCGI website by Mitch <http://finepointcgi.io/>`__
 - `GDScript website by Andrew Wilkes <https://gdscript.com>`__
-- `Godot Recipes by KidsCanCode <http://godotrecipes.com/>`__
+- `Godot Recipes by KidsCanCode <https://kidscancode.org/godot_recipes/>`__
```
Comment (skyace65) [inline community/tutorials.rst:53]:
```suggestion
- `Godot Recipes by KidsCanCode <https://kidscancode.org/godot_recipes/4.x/>`__
```

Given that this PR is for the master branch change it to this URL, not using the `4.x` makes the link default to 3.x recipes

**Review by skyace65 — CHANGES_REQUESTED** [review]

File: community/tutorials.rst (hunk 1 above)
Comment (cvlnomen) [reply community/tutorials.rst]: Done👍

**Review by skyace65 — APPROVED** [review]
Thanks! And congrats on your first merged PR!

**Pull request merged**